docs/content/concepts/partitions-schedules-sensors/backfills.mdx

```diff
@@ -7,41 +7,61 @@ description: Dagster supports data backfills for each partition or subsets of pa
 
 ## Overview
 
-Dagster supports data backfills for each partition or subsets of partitions. After defining a [partitioned job](/concepts/partitions-schedules-sensors/partitions), you can use backfills to submit runs for each partition in the set.
+Dagster supports data backfills for each partition or subsets of partitions. After defining a [partitioned asset or job](/concepts/partitions-schedules-sensors/partitions), you can use backfills to submit runs for each partition in the set.
 
 ---
 
 ## Launching Backfills
 
-### Using Dagit
+### Partitioned Asset Backfills
+
+You can open the backfill modal to launch backfills for a partitioned asset using the "Materialize" button on the [Asset detail page](https://docs.dagster.io/concepts/partitions-schedules-sensors/partitions#materializing-partitioned-assets).
+
+<Image
+  alt="backfills-launch-modal"
+  src="/images/concepts/assets/asset-backfill-partition-selection-modal.png"
+  width={3808}
+  height={2414}
+/>
+
```

Comment:
Probably also worth mentioning that backfills can be launched for a selection of differently partitioned assets, as long as the roots share the same partitioning